src/modules/gauging-stations/helpers.js

```diff
@@ -0,0 +1,22 @@
+const gaugingStationsCSVHeaders = [
+  'hydrology_station_id',
+  'station_reference',
+  'wiski_id',
+  'label',
+  'lat',
+  'long',
+  'easting',
+  'northing',
+  'grid_reference',
+  'catchment_name',
+  'river_name'
+];
+
+const getArraysFromCSV = Body => Body.toString()
```

Comment:
Could we use a 3rd party CSV parsing library here?  Elsewhere we have used `csv-parse`.  If anything is quoted in the CSV it quickly becomes a nightmare.